libvast/src/msgpack_table_slice.cpp

```diff
@@ -253,10 +253,11 @@ msgpack_map_view::value_type msgpack_map_view::at(size_type i) const {
 template <class FlatBuffer>
 msgpack_table_slice<FlatBuffer>::msgpack_table_slice(
   const FlatBuffer& slice, [[maybe_unused]] const chunk_ptr& parent,
-  const std::shared_ptr<arrow::RecordBatch>& batch) noexcept
+  const std::shared_ptr<arrow::RecordBatch>& batch, type schema) noexcept
```

Comment:
Is this param needed here? It seems unused, but i assume this is required for some generic code that passes all these parameters into different functions

Reply:
We no longer generate anything but Arrow v2 table slices, so this is essentially dead code. The parameter was just added to conform to a shared interface for construction as you assumed.